pkg/sfu/downtrack.go

```diff
@@ -1273,23 +1283,20 @@ func (d *DownTrack) handleRTCP(bytes []byte) {
 		return
 	}
 
+	sendPliOnce := func() {
+		if pliOnce {
+			d.lastPli.set(time.Now().UnixNano())
+			d.receiver.SendPLI(d.TargetSpatialLayer())
```

Comment:
I think target spatial layer is fine here. At least Chrome sends key frame on all layers irrespective of which layer sends PLI/FIR. Not sure about other browsers.

Reply:
Thank you for confirm behavior of chrome